exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/PlannerSettings.java

```diff
@@ -241,6 +244,10 @@ public long getInSubqueryThreshold() {
     return options.getOption(IN_SUBQUERY_THRESHOLD);
   }
 
+  public boolean isAnsiQuotesEnabled() {
```

Comment:
instead of a boolean, shouldn't we use a enum for the kind of quotes we support, so that we can extend it later?